index.html

```diff
@@ -35,15 +36,183 @@ <h3 class="motto" >
                     <li><a ><img class="twitter icons"src="images\Vector.svg" alt="twitter"></a></li>
                     <li><a><img class="linkedin icons" src="images\Linkedin icon@2x (1).png" alt="linkedin"></a></li>
                     <li><a><img class="vector-1 icons" src="images\Vector (1).svg" alt=""></a></li>
-                    <li><a><img class="vector-2 icons" src="images\Vector (2).svg" alt=""><a></a></li>
+                    <li><a><img class="vector-2 icons" src="images\Vector (2).svg" alt="github"><a></a></li>
                     <li><a><img class="vector-3 icons" src="images\Vector (3).svg" alt=""><a></li>
                 </ul>              
                
               </div>
             </div>
             
     </section>
-</main>
+  <!-- finish-mobile-version project starts here-->
+  <section class="works">
+<div class="card">
+  <a href=""><img class="img-1" src="images\Snapshoot Portfolio.png" alt=""></a>
+  <h2>
+    Tonic
+  </h2>
+  <div class="canopy-backend">
+  <ul>
+  <li class="canopy">CANOPY</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="back-end">Back End Dev</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="date">2015</li>
+  </ul>
+</div>
+<p class="daily-read">A daily selection of privately <br>personalized reads; no accounts or<br> sign-ups required.</p>
+
+<div class="dev-lang">
+<ul>
+  <li class="lang">HTML</li>
+  <li class="lang" id="lang">CSS</li>
+  <li class="lang" id="lang">JavaScript</li>
+
+</ul>
+</div>
+<button class="see-pr">See project</button>
+
+</div>
+<div class="card">
+  <a href=""><img class="img-1" src="images\Snapshoot Portfolio (1).png" alt=""></a>
+  <h2>
+    Multi-Post<br>Stories
+  </h2>
+  <div class="canopy-backend">
+  <ul>
+  <li class="canopy">CANOPY</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="back-end">Back End Dev</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="date">2015</li>
+  </ul>
+</div>
+<p class="daily-read">A daily selection of privately <br>personalized reads; no accounts or<br> sign-ups required.</p>
+
+<div class="dev-lang">
+<ul>
+  <li class="lang">HTML</li>
+  <li class="lang" id="lang">CSS</li>
+  <li class="lang" id ="lang">JavaScript</li>
+
+</ul>
+</div>
+<button class="see-pr">See project</button>
+
+</div>
+<div class="card">
+  <a href=""><img class="img-1" src="images\Snapshoot Portfolio (2).png" alt=""></a>
+  <h2>
+    Tonic
+  </h2>
+  <div class="canopy-backend">
+  <ul>
+  <li class="canopy">CANOPY</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="back-end">Back End Dev</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="date">2015</li>
+  </ul>
 </div>
```

Comment:
- [Optional] Kindly ensure you stick to design as much as possible, this includes, padding, margins, font sizes, font weights, fonts and colors. Please improve on the following.
### Your design
![image](https://user-images.githubusercontent.com/94551484/227738873-b91cb557-9633-4b88-ae3f-35de4dc6f3a3.png)
![image](https://user-images.githubusercontent.com/94551484/227738939-086c6a3a-f70e-4c9a-8b37-84b8926a0214.png)

### FIgma
![image](https://user-images.githubusercontent.com/94551484/227738972-58de7e0e-9d4f-4e1f-88d6-b3c631224748.png)